### PR TITLE
remove irrelevant comment

### PR DESCRIPTION
**Pull request by leahecole**
Thanks @adlersantos for noticing that these comments are relevant for the kuberenetesPodOperator but not the GKE operator!!

### CODE REVIEW

**Review by busunkim96 — APPROVED** [review]

File: composer/airflow_1_samples/gke_operator.py [hunk 1]
```diff
@@ -85,11 +85,7 @@
         # entrypoint is used. The cmds parameter is templated.
         cmds=["echo"],
         # The namespace to run within Kubernetes, default namespace is
-        # `default`. There is the potential for the resource starvation of
-        # Airflow workers and scheduler within the Cloud Composer environment,
-        # the recommended solution is to increase the amount of nodes in order
-        # to satisfy the computing requirements. Alternatively, launching pods
-        # into a custom namespace will stop fighting over resources.
+        # `default`.
```
Comment (dandhlee) [inline composer/airflow_1_samples/gke_operator.py:88]:
```suggestion
        # `default`.
```
Trailing whitespace, lint complaining :S

File: composer/workflows/gke_operator.py [hunk 2]
```diff
@@ -76,11 +76,7 @@
         # entrypoint is used. The cmds parameter is templated.
         cmds=["echo"],
         # The namespace to run within Kubernetes, default namespace is
-        # `default`. There is the potential for the resource starvation of
-        # Airflow workers and scheduler within the Cloud Composer environment,
-        # the recommended solution is to increase the amount of nodes in order
-        # to satisfy the computing requirements. Alternatively, launching pods
-        # into a custom namespace will stop fighting over resources.
+        # `default`.
```
Comment (dandhlee) [inline composer/workflows/gke_operator.py:79]:
```suggestion
        # `default`.
```
ditto

**Review by dandhlee — CHANGES_REQUESTED** [review]

**Comment by leahecole** [comment]
omg serves me right for trying to submit a PR during a meeting. 🤦🏻‍♀️  I'll fix that lint

**Review by dandhlee — APPROVED** [review]

**Review by adlersantos — APPROVED** [review]
Thanks @leahecole!

**Pull request merged**